esi_leap/objects/flocx_market_client.py

```diff
@@ -63,3 +64,65 @@ def send_offer(self, data):
                 r.status_code,
                 r.text)
         return r.status_code
+
+    def get_contracts(self):
+        url = CONF.flocx_market.endpoint_override + \
+        '/offer_contract_relationship?status=unretrieved'
+        r = self.get(url)
+        if r.status_code != 200:
+            LOG.warning(
+                "Failed to retrieve contracts from flocx-market. Got HTTP %s: %s",
+                r.status_code,
+                r.text)
```

Comment:
Shouldn't we `return` at this point, rather than falling through the rest of the function?

Reply:
Ah, yep! I've changed it to return [] when r.status_code != 200
https://github.com/DanNiESh/esi-leap/blob/60f70babd525ad00915ed879c2f59fc4bf6aee5c/esi_leap/common/flocx_market.py#L84